doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,137 @@ Topics include:
+
+<a name="9.6.1.1"></a>
+#### 9.6.1.1. What to observe
+
+Typically, when it comes to data collection, three questions arise:
+1. Which data to collect?
+2. Where to send the data?
+3. Which protocol/interface/format to use?
+
+<a name="9.6.1.2"></a>
+#### 9.6.1.2. What data to collect
+
+Assessment on which data to collect should start by iterating over the physical and virtual infrastructure components: 
+
+* Network Services across sites and tenants 							
+* Virtualized functions per site and tenant
+* Individual Virtual Machines and Containers
+* Virtualization infrastructure components
+* Physical servers (compute) and network elements
+* Toolservers with their applications (DNS, Identity Management, Zero Touch Provisioning, etc) 
+* Cabling 
+
+<a name="9.6.1.3"></a>
+#### 9.6.1.3. Data categories
+
+There are three main observability categories: metrics, alarms and logs:
+
+1. **Metrics** or telemetry report counters and gauge levels and can either be pulled periodically e.g. via SNMP or REST, or pushed as streams using gRPC, NETCONF, which receivers registered for certain sensors, or by registering as a publisher to a Kafka bus. These messages must be structured in order to get parsed successfully.
+
+2. **Alarms** indicate state changes, categorized by severity, often with a description of what just
+hap- pened. Most common transport protocol is SNMP (traps and inform messages). They
+are generated by physical network elements as well as logical devices. In addition, they can
+also be generated by monitoring applications in form of configured thresholds or dynamic by
+means of Machine Learning (ML) algorithms - generally, they are describing anomalies.
+
+3. **Logs** messages are generated by most physical devices (compute and network) and virtual
+applications and transported over SYSLOG and tend to come in high volumes.
+
+4. **Traces** are end-to-end signalling messages (events) created to fullfil execution of requests on
+the distributed system services. OTHER WORDS: Traces are all action points executed in
+order to provide response to the request set to the distributed system service. Even the call
+can be thought of as a request which starts by INVITE message of the SIP protocol.
+
+<a name="9.6.1.4"></a>
+#### 9.6.1.4. Where to send the data
+
+If the sources of observability data have to send their data towards particular specific destinations, then it is a concept with high dependency and extremely prone to errors, especially in case of configuration changes. In order to keep it clean and elegant, data sources should be completely independent of the data receivers; i.e. data sources must not be impacted with any change in the data receivers.
```

Comment:
```suggestion
If the observability data have to be sent from their sources (or Producers) to specific destinations (or Consumers), then this creates a high degree of dependency between Producers and Consumers, and is extremely prone to errors, especially in case of configuration changes. Ideally, the data Producers must not be impacted with any change in the data Consumers and vice versa.
```

Reply:
Thanks, this sounds much more elegant and clear! I would accept copletely.

Reply:
Fixed